test/modules/aws-rds-integration.ts

```diff
@@ -411,7 +440,7 @@ describe('RDS Integration Testing', () => {
       query(`
       SELECT invoke_rds(
         'createDBCluster',
-        '{"Engine": "aurora", "AvailabilityZone": "us-west-2a", "DBClusterIdentifier": "${prefix}hidden", "MasterUserPassword": "dontcare", "MasterUsername": "dontcare"}',
+        '{"Engine": "aurora", "AvailabilityZone": "${availabilityZone}", "DBClusterIdentifier": "${prefix}hidden", "MasterUserPassword": "dontcare", "MasterUsername": "dontcare"}',
```

Comment:
I don't get how this can fix things

Reply:
I'm not 100% sure, to be honest, but it was the main weird thing I saw... 

The issue is that in order to create the aurora cluster, we are calling the `aws_sdk` directly and it is responding with a wrong engine version error. My suspicion is that the fixed `availability_zone` might be interfering with the command to pick the right engine version for some regions (since the region is changing dynamically).

Reply:
yes